src/App.jsx

```diff
@@ -19,7 +20,7 @@ function App() {
 				<Route path="/" element={<Home />}></Route>
 				<Route path="/topics" element={<Topics />}></Route>
 				<Route path="/articles" element={<Articles />}></Route>
-				<Route path="/articles/:article_id" element={<Article />}></Route>
+				<Route path="/articles/:article_id" element={<Article comments={Comments} />}></Route>
```

Comment:
it looks like your passing in the comments component here but not using this prop and rather using the imported comment component inside the article component - in which case this props is not needed